templates/idrive.xml

```diff
@@ -0,0 +1,26 @@
+<?xml version="1.0"?>
+<Container version="2">
+  <Name>iDrive</Name>
+  <Repository>tooliload/unraid-idrive</Repository>
+  <Registry>https://hub.docker.com/r/tooliload/unraid-idrive</Registry>
+  <Network>bridge</Network>
+  <MyIP/>
+  <Shell>sh</Shell>
+  <Privileged>false</Privileged>
+  <Support/>
+  <Project/>
+  <Overview/>
+  <Category>Backup: Cloud:</Category>
+  <WebUI/>
+  <TemplateURL>https://raw.githubusercontent.com/tooliload/docker-templates/master/templates/unraid-idrive.xml</TemplateURL>
+  <Icon>https://raw.githubusercontent.com/tooliload/docker-templates/master/templates/icons/idrive.pngg</Icon>
+  <ExtraParams></ExtraParams>
+  <PostArgs/>
+  <CPUset>0</CPUset>
+  <DonateText/>
+  <DonateLink/>
+  <Requires/>
```

Comment:
Remove unused fields. <https://selfhosters.net/docker/templating/templating/#114-shave-off-the-xml>